WordPress/Classes/Networking/BlogServiceRemoteREST.m

```diff
@@ -217,6 +252,15 @@ - (NSDictionary *)mapPostFormatsFromResponse:(id)response
     }
 }
 
+- (RemotePostType *)remotePostTypeWithDictionary:(NSDictionary *)json
+{
+    RemotePostType *postType = [[RemotePostType alloc] init];
```

Comment:
Just a thought regarding the non-null / nullable thing, would it be better to specify non-null and have an initializer accepting the name, label, etc?